### PR TITLE
FIX: column names for AK

### PR DESCRIPTION
**Pull request by tlyon3**

### CODE REVIEW

File: src/cmdc_tools/datasets/official/AK/__init__.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
 def _find_col_by_prefix(df, col_prefix):
     cols = [x for x in list(df) if x.lower().startswith(col_prefix.lower())]
-    assert len(cols) == 1
```
Comment (sglyon) [inline src/cmdc_tools/datasets/official/AK/__init__.py:35]: Hmm are we confident this is correct? What are the two columns that are found?

**Review by sglyon — CHANGES_REQUESTED** [review]
Thanks for taking a look @tlyon3 !

just one question

File: src/cmdc_tools/datasets/official/AK/__init__.py (hunk 1 above)
Comment (tlyon3) [reply src/cmdc_tools/datasets/official/AK/__init__.py:35]: Yes I looked at it. They are `Community` and `Community\nCases`

Comment (sglyon) [reply src/cmdc_tools/datasets/official/AK/__init__.py:35]: What is the difference between the two in terms of data? I see that later in the PR you started using the names with `\n` in them.

**Comment by cc7768** [comment]
Alaska has their data in an ArcGIS dashboard and we can access it in a more clean way than we currently are.

Is there any opposition to refactoring this code to use the ArcGIS class/endpoint?

**Comment by sglyon** [comment]
no opposition. sounds good to me

**Comment by cc7768** [comment]
Closing in favor of #62

**Pull request closed without merging**